src/app/utils/keycloak/keycloakUtils.ts

```diff
@@ -42,8 +42,9 @@ export const initKeycloak = async (
       // Then force a token refresh to check if the refresh token is actually valid
       await rk.updateToken(-1);
       return rk;
-    } catch {
+    } catch (e) {
       clearRefreshToken();
+      console.log("Triggering MASSSO logout because of error with existing refresh token: " + e);
```

Comment:
I think we should move to console.debug for that values. Those will still be printed as we do not have any logger configured in UI but when we do then we will minimalize work required etc.


THis is natively supported in browsers without shim
https://developer.mozilla.org/en-US/docs/Web/API/console/debug